youtube_dl/options.py

```diff
@@ -502,6 +502,11 @@ def _scrub_eq(o):
         '--sleep-interval', metavar='SECONDS',
         dest='sleep_interval', type=float,
         help='Number of seconds to sleep before each download.')
```

Comment:
Description should be clarified for the case when using with `--max-sleep-interval`.